test/helpers/table.js

```diff
@@ -0,0 +1,28 @@
+import { GetItemCommand } from '@aws-sdk/client-dynamodb'
+import { marshall, unmarshall } from '@aws-sdk/util-dynamodb'
+import pRetry from 'p-retry'
+
+/**
+ * @param {import('@aws-sdk/client-dynamodb').DynamoDBClient} dynamo
+ * @param {string} tableName
+ * @param {object} key
+ */
+export async function waitForTableItem (dynamo, tableName, key) {
```

Comment:
Nit: I think `wait` is misleading (makes me think it waits for an event to occur), it would be more accurate to call it `poll` IMO

```suggestion
export async function pollTableItem (dynamo, tableName, key) {
```